crux/src/Crux/Config/Common.hs

```diff
@@ -442,6 +447,12 @@ cruxOptions = Config
       , Option [] ["no-unsat-cores"]
         "Disable computing unsat cores for successful proofs"
         $ NoArg $ \opts -> Right opts { unsatCores = False }
+      
+      , Option "n" ["get-abducts"]
```

Comment:
It would be worth mentioning that:

1. `crux-llvm` now supports CVC5 as a solver, and
2. There is now a new `get-abducts` option

In the [`crux-llvm` `CHANGELOG` file](https://github.com/GaloisInc/crucible/blob/cca9221b70f22b8d77ad716cf6ccb57156f00785/crux-llvm/CHANGELOG.md).